pengwin-setup.d/x410.sh

```diff
@@ -1,8 +1,18 @@
 #!/bin/bash
 
-if (whiptail --title "X410" --yesno "Would you like to view a link to X410 (recommended) on the Microsoft Store?" 8 80) then
-	echo "Running $ wslview <link>"
-	wslview https://afflnk.microsoft.com/c/1291904/433017/7593?u=https%3A%2F%2Fwww.microsoft.com%2Fen-us%2Fp%2Fx410%2F9nlp712zmn9q%23activetab%3Dpivot%3Aoverviewtab
+# check if x410 exists
+if (powershell.exe Get-Command x410); then
+        echo "Configuring X410 to start on Pengwin launch"
+        sudo bash -c 'cat > /etc/profile.d/02-x410.sh' << EOF
+#!/bin/bash
+
+(powershell.exe x410.exe /wm /public &> /dev/null &)
```

Comment:
Avoid using powershell at startup scripts because it is slow. I think this can be replaced by calling directly the program or using cmd.exe